terraform/snowflake/modules/policies/variables.tf

```diff
@@ -1,3 +1,9 @@
+variable "environment" {
+  description = "Environment suffix"
+  type        = string
+  default     = "PRD"
+}
```

Comment:
This is meant for account-level things. Why do we have an `environment` variable?